### ISSUE TITLE
what the difference between the private version from the free version

### ISSUE THREAD

**Issue by RandomNonDev** [piyushbest/TheBridge#1]

**Comment by piyushbest** [comment]
i will start work on private one from tomorrow,
private version include this features-

Can set goal pos(in free version you can only goal by moving on Red Dazed Terracotta and Blue Dazed Terracotta)

Custom Scoreboard
LeaderBoard
Kits(not included on free one)

Modes(1v1, 2v2, squad, 8v8)
can set how much goal to win(5 goal on free ver)
and much more(you can suggest)


for now paid is cheap you can buy (you will be updated whenever new version come, you dont have to buy again)

thats all

**Comment by piyushbest** [comment]
also english is not my main language so sorry for my english

**Comment by JoeXerus** [comment]
Can I help in paid also ? I was out home so I don't updated anythìng

**Comment by JoeXerus** [comment]
Piyush your Indian?

**Comment by piyushbest** [comment]
Yes joexerus i am indian, and i know your also indian, i saw your comment on video

**Comment by piyushbest** [comment]
i dont need help :) already done
